libcnb-data/src/launch.rs

```diff
@@ -184,29 +182,27 @@ pub struct ProcessBuilder {
 impl ProcessBuilder {
     /// Constructs a new `ProcessBuilder` with the following defaults:
```

Comment:
(Optional) I wonder if we need a quick mention in this rustdocs about why `command` is a vec, and how it differs from the args? As in, to avoid the scenario where a new user of libcnb doesn't see `.arg()` / `.args()` and (quite reasonably) presumes that `command` being an array means all args should just be added there?

Reply:
I have a hard time to come up with something here. I agree that `args` is now confusing but that's on the upstream spec. It's not super well explained there either. I will mark this a resolved and merge, we can add something in a separate PR.